app/uploaders/alert_image_uploader.rb

```diff
@@ -2,6 +2,8 @@ class AlertImageUploader < ApplicationUploader
   include CarrierWave::MiniMagick
   include ::CarrierWave::Backgrounder::Delay
 
+  alias stolen_record model
```

Comment:
I feel like `alias_method :stolen_record, :model` reads a little bit better. Open to different opinion though

Reply:
Agreed, but I think the semantic difference dominates for me — `alias` is lexically scoped, which is more restrictive / clearer in this case. (It was the style guide that switched my default on this... that said, it's a weakly set default):

> Prefer alias when aliasing methods in lexical class scope as the resolution of self in this context is also lexical, and it communicates clearly to the user that the indirection of your alias will not be altered at runtime or by any subclass unless made explicit.

> https://github.com/rubocop-hq/ruby-style-guide#alias-method-lexically